pyvips/vimage.py

```diff
@@ -324,11 +324,11 @@ def __exit__(self, type, value, traceback):
     def __getitem__(self, arg):
         if isinstance(arg, slice):
             i = 0
-            if arg.start != None:
+            if arg.start is not None:
```

Comment:
With commit https://github.com/jcupitt/pyvips/pull/5/commits/a81035fb75250372355dc23c35d81a6f6ebf82b6 the whole package passes `pycodestyle` (formerly called `pep8`) and `PyFlakes` (a code analyzer). Tested with `flake8`, which combines `Pyflakes` with style checks against `PEP 8`.

I also removed the `package_index` from ` __init__.py`. (it was solvable with import and referencing the objects using the module name). Running `nosetests` seems to be a bit faster now.